code/game/turfs/open/openspace.dm

```diff
@@ -21,6 +21,7 @@ GLOBAL_DATUM_INIT(openspace_backdrop_one_for_all, /atom/movable/openspace_backdr
 	CanAtmosPassVertical = ATMOS_PASS_YES
 	baseturfs = /turf/open/openspace
 	intact = FALSE //this means wires go on top
+	mouse_opacity = MOUSE_OPACITY_TRANSPARENT
 	//mouse_opacity = MOUSE_OPACITY_TRANSPARENT
```

Comment:
uwu?

Reply:
Else you can't hover over things a floor below you because open space is layered on top, me and moth couldn't find any bugs with this

Reply:
(might be good to test though)

Reply:
also i never saw that commented out mouse transparency thing wtf...ill interrogate kevinz